### PR TITLE
First draft May newsletter

### PR DESCRIPTION
**Pull request by bethaitman**
Ready for review!

### CODE REVIEW

File: docs/blog/newsletter-may-2019.rst [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+Write the Docs Newsletter – May 2019
+####################################
+
+Hello everyone! It's Beth and the Write the Docs newsletter team here, with May's edition full of news and collected wisdom from the community Slack.
+
+This month's biggest announcement is `the dates for Write the Docs Australia </conf/australia/2019/>`__. It'll be 14-15 November 2019, in Sydney. Save the date!
+
+To the nearer future, `the Portland conference </conf/portland/2019/news/schedule-tickets/>`__ is coming up on 19 May. For those who can't make it, you'll be pleased to hear that we'll be live-streaming the conference again this year! `www.writethedocs.org/conf/portland/2019/livestream </conf/portland/2019/livestream/>`__ is the place to watch.
+
+And if you're planning to attend one of our European conferences, now's a great time to buy tickets, which are on sale now for `Vilnius </conf/vilnius/2019/tickets/>`__ and `Prague </conf/prague/2019/tickets/>`__. We're delighted to tell you that `diversity tickets are available for Prague </conf/prague/2019/news/cfp-diversity-tickets/#diversity-tickets-initiative>`__!
```
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst]: We're also delighted... (add "also" maybe?)

File: docs/blog/newsletter-may-2019.rst [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+Finally, just a reminder: If you spot something on Slack and you think, "That was interesting/useful/cool, we should save that", please go ahead and pop the ``:suggest-for-newsletter:`` emoji on it. That'll bring it to our attention, so we can write it up into a newsletter article and save for posterity!
```
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst]: ...save it for posterity! (add "it")

File: docs/blog/newsletter-may-2019.rst [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+* `CSAT - what’s that? </videos/na/2016/csat-what-s-that-betsy-roseberg/>`_
+* `Feedback handling, community wrangling, panhandling </videos/eu/2016/feedback-handling-community-wrangling-panhandling-chris-mills/>`_
+* `You have already succeeded 2.0 </videos/eu/2017/you-have-already-succeeded-design-critique-guidelines-make-feedback-easier-2-0-christy-lutz/>`_
+
+------------------------------------
+Tips and tricks for producing videos
+------------------------------------
+
+Videos can be a great way of educating users, but it's not always easy to do them well, especially if you haven't worked on video before. Here are some tips from a recent community community:
```
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst]: "community" is repeated at the end of this intro sentence

File: docs/blog/newsletter-may-2019.rst [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+* Keep in mind that videos are not a "quick project": they can be very time consuming.
```
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst]: hyphenate "time-consuming" maybe?

File: docs/blog/newsletter-may-2019.rst [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+* Create a pipeline for video creation and production that suits your organization's specific needs.
+* Consider developing a library of assets for videos and commit to maintaining it.
+* Be sure to take up-front expenses into account, such as editing software or a higher quality microphone.
```
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst]: again hyphenate "higher-quality" as a modifier?

File: docs/blog/newsletter-may-2019.rst [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+* If you don't have the resources to produce full-fledged tutorial videos, consider shorter videos (five minutes or less) that serve as an introduction to draw the audience into the documentation.
+* Collaborate with other teams! UX, Customer Support, Training, and other teams can all be valuable resources for videos.
+* Many community members suggested that docs teams should only commit to video content if they have a dedicated person, or even team, to work on videos.
+
+If you've got any more questions, head on over to `#video <https://writethedocs.slack.com/messages/video/>`_!
+
+------------------------------------------------------
+The variable fortunes of variable placeholders in docs
+------------------------------------------------------
```
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst:58]: love this heading!

File: docs/blog/newsletter-may-2019.rst [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+
+In code examples, you often need to put in placeholders for values that users must provide. But no matter what a hapless writer tries, it seems inevitable that users won't realise they need to substitute in their own data. Here's a summary of the options that our documentarians have provided for their readers:
```
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst]: I think "in" is redundant with "substitute" -- "...substitute their own data."

File: docs/blog/newsletter-may-2019.rst [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+Developer to documentarian ratio
+--------------------------------
+
+This month, one community member was looking for ideas for when your company asks "Why aren't our docs as great as <other doc site>?" Especially if you don't have the resources you'd need to make something equivalent.
+
+One approach is to find out how many people the other company puts on docs. For example, many folks find their docs being compared to Stripe's; but community members reported that Stripe's site is custom-built, with dedicated engineers as well as writers (plural, of course).
+
+You can also try breaking down the work that's required to get there, and use that to ask for more people (or for the pressure to be taken off). With positive phrasing: "I'd love to do that! Now, here are the resources we'll need to make it happen."
+
+But how many people *do* you need? The ratio of writers to developers varies hugely. `This report <https://www.infomanagementcenter.com/publications/e-newsletter/cidm-enews-03-17/another-look-at-writer-to-developer-staffing-ratios/>`__ had found that between 1:7 and 1:12 was fairly standard, but community members reported ratios as high as 1:51.
```
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst]: bit odd to say the report found -- maybe "One survey of articles and forum posts found that..." instead?

File: docs/blog/newsletter-may-2019.rst [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+
+It varies because needs vary: a developer team focused on performance improvements won't generate much doc work, where two working on big UI changes might create lots. Even with a great writer-to-dev ratio, if the team is building many complex features, writers may still struggle to keep up. Some thought that the ratio of writers to UX designers or product managers might match docs work generated more closely.
```
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst]: "whereas" instead of "where"?

File: docs/blog/newsletter-may-2019.rst [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+
+There's further discussion on the right ratio in `this blog post <http://www.agiledocumentation.co.uk/2016/04/what-is-optimal-writerdeveloper-ratio.html>`__, and `this post on Medium <https://medium.com/ixda-berlin/staffing-ratios-finding-the-right-balance-between-pm-ux-and-engineering-in-your-team-12ada861a1d0>`__.
+
+---------
+Job posts
+---------
+
+If you're in Seattle, this is your month!
```
Comment (hillaryfraley) [inline docs/blog/newsletter-may-2019.rst:92]: :D love it!

File: docs/blog/newsletter-may-2019.rst [hunk 11]
```diff
@@ -0,0 +1,134 @@
...
+
+This month's biggest announcement is `the dates for Write the Docs Australia </conf/australia/2019/>`__. It'll be 14-15 November 2019, in Sydney. Save the date!
+
+To the nearer future, `the Portland conference </conf/portland/2019/news/schedule-tickets/>`__ is coming up on 19 May. For those who can't make it, you'll be pleased to hear that we'll be live-streaming the conference again this year! `www.writethedocs.org/conf/portland/2019/livestream </conf/portland/2019/livestream/>`__ is the place to watch.
+
+And if you're planning to attend one of our European conferences, now's a great time to buy tickets, which are on sale for `Vilnius </conf/vilnius/2019/tickets/>`__ and `Prague </conf/prague/2019/tickets/>`__. We're delighted to tell you that `diversity tickets are available for Prague </conf/prague/2019/news/cfp-diversity-tickets/#diversity-tickets-initiative>`__!
+
+Finally, just a reminder: If you spot something on Slack and you think, "That was interesting/useful/cool, we should save that", please go ahead and pop the ``:suggest-for-newsletter:`` emoji on it. That'll bring it to our attention, so we can write it up into a newsletter article and save for posterity!
```
Comment (clairealyssa) [inline docs/blog/newsletter-may-2019.rst]: Do we want to somehow say here that it brings it to consideration for the newsletter? Things marked with the emoji may not be 1:1 with newsletter articles.

File: docs/blog/newsletter-may-2019.rst [hunk 12]
```diff
@@ -0,0 +1,134 @@
...
+
+Speaking of which: what interesting things have come up this month?
```
Comment (clairealyssa) [inline docs/blog/newsletter-may-2019.rst]: Should this be a colon or a comma? Just a small nitpick.

File: docs/blog/newsletter-may-2019.rst (hunk 4 above)
Comment (clairealyssa) [reply docs/blog/newsletter-may-2019.rst]: Even though I originally wrote it without the hyphen, looking at it here, I have turned team pro-hyphen.

**Review by clairealyssa — COMMENTED** [review]
Thank you, Beth!

File: docs/blog/newsletter-may-2019.rst (hunk 1 above)
Comment (bethaitman) [reply docs/blog/newsletter-may-2019.rst]: agreed!

File: docs/blog/newsletter-may-2019.rst (hunk 3 above)
Comment (bethaitman) [reply docs/blog/newsletter-may-2019.rst]: fixed!

File: docs/blog/newsletter-may-2019.rst (hunk 4 above)
Comment (bethaitman) [reply docs/blog/newsletter-may-2019.rst]: +1, changed

File: docs/blog/newsletter-may-2019.rst (hunk 7 above)
Comment (bethaitman) [reply docs/blog/newsletter-may-2019.rst]: agreed, taken it out

File: docs/blog/newsletter-may-2019.rst (hunk 8 above)
Comment (bethaitman) [reply docs/blog/newsletter-may-2019.rst]: :+1:

File: docs/blog/newsletter-may-2019.rst (hunk 9 above)
Comment (bethaitman) [reply docs/blog/newsletter-may-2019.rst]: 👍

File: docs/blog/newsletter-may-2019.rst (hunk 11 above)
Comment (bethaitman) [reply docs/blog/newsletter-may-2019.rst]: Yeah, might be worth making it clear that we don't write up everything. I'll adjust.

File: docs/blog/newsletter-may-2019.rst (hunk 12 above)
Comment (bethaitman) [reply docs/blog/newsletter-may-2019.rst]: changed

File: docs/blog/newsletter-may-2019.rst [hunk 13]
```diff
@@ -0,0 +1,134 @@
...
+
+####################################
+Write the Docs Newsletter – May 2019
+####################################
+
+Hello everyone! It's Beth and the newsletter team here, with May's edition full of news and collected wisdom from the community Slack.
+
+This month's biggest announcement is `the dates for Write the Docs Australia </conf/australia/2019/>`__. It'll be 14-15 November 2019, in Sydney. Save the date!
+
+To the nearer future, `the Portland conference </conf/portland/2019/news/schedule-tickets/>`__ is coming up on 19 May. For those who can't make it, you'll be pleased to hear that we'll be live-streaming the conference again this year! `www.writethedocs.org/conf/portland/2019/livestream </conf/portland/2019/livestream/>`__ is the place to watch.
```
Comment (plaindocs) [inline docs/blog/newsletter-may-2019.rst]: > to the nearer future

 this reads weird at first glance.

**Comment by bethaitman** [comment]
Still waiting on an approval, if anyone cares to?

**Review by plaindocs — APPROVED** [review]
Excellent work!

File: docs/blog/newsletter-may-2019.rst (hunk 13 above)
Comment (bethaitman) [reply docs/blog/newsletter-may-2019.rst]: I can rephrase

**Pull request merged**